### PR TITLE
Atom radio button component

### PR DESCRIPTION
**Pull request by juanmaguitar**
- ⚙︎Task → https://jira.scmspain.com/browse/SUIC-133
- 🌎Demo Online → https://sui-components-atom-radio-button-component.now.sh/workbench/atom/radioButton/demo

### CODE REVIEW

File: components/atom/radioButton/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$w-atom-radiobutton: 15px !default;
```
Comment (jcger) [inline components/atom/radioButton/src/index.scss]: all hardcoded values should be tokens

File: components/atom/radioButton/src/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomRadioButton'
+const CLASS_HIDDEN = `is-hidden`
+
+const ERROR_STATES = {
```
Comment (giodelabarrera) [inline components/atom/radioButton/src/index.js]: I get the feeling of how you use within the components this object is more like response states. If so, I think that for greater readability it should be `RESPONSE_STATUS`

File: components/atom/radioButton/src/index.scss (hunk 1 above)
Comment (juanmaguitar) [reply components/atom/radioButton/src/index.scss]: All values replaced by the proper token but this one:
$bw-atom-radiobutton-checked: 6px !default;

As there is no token w/ this value and I need exactly this value to emulate the effect reflected in the specification

File: components/atom/radioButton/src/index.js (hunk 2 above)
Comment (juanmaguitar) [reply components/atom/radioButton/src/index.js]: Removed ERROR_STATES logic as this component doesn't really need to be aware of this (the specification doesn't reflect any behaviour in this level regarding errors)

**Review by kikoruiz — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Pull request merged**